packages/mui-material-next/src/ButtonGroup/ButtonGroup.test.js

```diff
@@ -0,0 +1,257 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { createRenderer, describeConformance, screen } from '@mui-internal/test-utils';
+import ButtonGroup, { buttonGroupClasses as classes } from '@mui/material-next/ButtonGroup';
+import Button, { buttonClasses } from '@mui/material-next/Button';
+import ButtonGroupContext from './ButtonGroupContext';
+import { CssVarsProvider, extendTheme } from '../styles';
+
+describe('<ButtonGroup />', () => {
```

Comment:
To fix the unit test, you need to kind of mock `window.matchMedia` like [this](https://github.com/mui/material-ui/blob/master/packages/mui-material-next/src/ButtonBase/ButtonBase.test.tsx#L46-L56)

And as part of this, the test should be converted to TS as well ~